contracts/staking/SimpleStaking.sol

```diff
@@ -111,7 +109,7 @@ contract SimpleStaking is AbstractGoodStaking, StakingToken {
 		}
 		_burn(msg.sender, _amount); // burn their staking tokens
 		_decreaseProductivity(msg.sender, _amount);
-		fm.mintReward(address(iToken), msg.sender); // send rewards to user
+		fm.mintReward(nameService.getAddress("CDAI"), msg.sender); // send rewards to user and use cDAI address since reserve in cDAI
```

Comment:
what? simple staking is a generic contract

Reply:
Yes but reserve just in cDAI and this parameter needed to update RR so must be cDAI